From 7131b80b8627dde74940bf4f0c23dee87f4f7187 Mon Sep 17 00:00:00 2001
From: drabina <krzysiek.drobinski@gmail.com>
Date: Fri, 27 Jan 2023 23:50:31 +0100
Subject: [PATCH] openssl: Don't fetch internal openssl.cnf when building for
 shared

---
 node.gyp | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/node.gyp b/node.gyp
index aae252e6..ba3cb49f 100644
--- a/node.gyp
+++ b/node.gyp
@@ -434,18 +434,6 @@
               'opensslconfig_internal': '<(obj_dir)/deps/openssl/openssl.cnf',
               'opensslconfig': './deps/openssl/nodejs-openssl.cnf',
            },
-           'actions': [
-             {
-               'action_name': 'reset_openssl_cnf',
-               'inputs': [ '<(opensslconfig)', ],
-               'outputs': [ '<(opensslconfig_internal)', ],
-               'action': [
-                 '<(python)', 'tools/copyfile.py',
-                 '<(opensslconfig)',
-                 '<(opensslconfig_internal)',
-               ],
-             },
-           ],
          }],
       ],
     }, # node_core_target_name
-- 
2.34.1

